lib/feature-config/feature/filter.rb

```diff
@@ -3,6 +3,11 @@ def self.build_filters(attributes)
     attributes.map { |name, options| class_for(name).new(options) }
   end
 
+  def self.class_for(filter_name)
+    "#{name}/#{filter_name}".classify.constantize
+  end
+  private_class_method :class_for
```

Comment:
I can also use the new syntax of ruby 2.2.x .

```rb
private_class_method  def self.class_for(filter_name)
  # code
end
```

but I like the current implemented way.